docs/docs/reference/built-in-components/gatsby-plugin-image.md

```diff
@@ -0,0 +1,185 @@
+
+For `fixed` and `constrained` images, you can also optionally pass either a `width` or `height`, and it will use that to calculate the other dimension. For example, if you pass `width={800} aspectRatio={4/3}` then `height` will be set to the width divided by the aspect ratio: so `600`. Passing `1` as the aspectRatio will crop the image to a square. If you don't pass a width or height then it will use the source image's width.
+
+For `fullWidth` images you don't specify width or height as it resizes to fit the screen width. Passing `aspectRatio` will crop the image if needed, and the height will scale according to the width of the screen. For example, if you set the aspectRatio to `16/9` then when the image is displayed full width on a screen that is 1280px wide, the image will be 720 pixels high.
+
+> There are several advanced options that you can pass to control the cropping and resizing behavior. For more details, see the [`transformOptions`](#transformoptions) reference.
+
+### `placeholder`
+
+Gatsby image components are lazy-loaded by default, which means that if they are offscreen they are not loaded by the browser until the come into view. To ensure that the layout does not jump around, a placeholder is displayed before the image loads. There are three types of placeholder that you can choose
+
+| Placeholder    | Component prop value | Resolver prop value | Description                                                                                                                                                                        |
+| -------------- | -------------------- | ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Dominant color | `"dominantColor"`    | `DOMINANT_COLOR`    | The default placeholder. This calculates the dominant color of the source image and uses it as a solid background color.                                                           |
+| Blurred        | `"blurred"`          | `BLURRED`           | This generates a very low-resolution version of the source image and displays it as a blurred background.                                                                          |
+| Traced SVG     | `"tracedSVG"`        | `TRACED_SVG`        | This generates a simplified, flat SVG version of the source image, which it displays as a placeholder. This works well for images with simple shapes or that include transparency. |
+| None           | `"none"`             | `NONE`              | No placeholder. You can use the background color option to set a static background if you wish.                                                                                    |
+
+### `formats`
+
+_Default component prop value: `["auto", "webp"]`. Default resolver prop value: `[AUTO, WEBP]`_
```

Comment:
Was this line left here intentionally? Might be able to get rid of it, since it's in the table up above. (But I also don't mind having the duplication here - my callout is mostly just that this is the only option that still has a default listed this way.)